tools/mbpp/fix_v011.py

```diff
@@ -0,0 +1,43 @@
+def check_id(data, n, task_id):
+    assert data[n]["task_id"] == task_id
+
+
+def fix(data):
+    # fix: https://github.com/evalplus/evalplus/issues/210
+
+    check_id(data, 215, "Mbpp/459")
+    data[215]["canonical_solution"] = data[215]["canonical_solution"].replace(
```

Comment:
Prefer directly assigning the code content over doing subpart editing esp. when the code is small. ;)